src/__acceptance_tests__/popover-ssr-acceptance-test.tsx

```diff
@@ -1,6 +1,5 @@
 import {openSSRPage} from '../test-utils';
 
 test('ssr popover', async () => {
-    // don't check hydration mismatch because popovers are only rendered client side inside a portal
```

Comment:
We needed this because popover was being opened by default. I think that was wrong, because you don't actually want all these popups to show unless the user clicks on them.